### ISSUE TITLE
Invoice filters don't appear to work

### ISSUE THREAD

**Issue by blairjo** [XeroAPI/xero-node#339]
**SDK you're using (please complete the following information):**
 - Version [e.g. 4.1.1]

**Describe the bug**
When I call the getInvoices() function and pass 1 contactId (inside an array) and an array of statuses and a page, the result is not limited to the contact / status values - I just get every invoice.

**To Reproduce**

```
let contactIds = ['bob'];
let status = ['PAID','AUTHORISED'];

let result = await xero.accountingApi.getInvoices(xero.tenantIds[0], undefined, undefined, undefined, undefined, undefined, contactIds, status, page)

```

All invoices are returned. 

**Expected behavior**
Only invoices for the passed contact id and statuses should be returned

**Comment by SidneyAllen** [comment]
@SerKnight - can you make some test calls using xero-node and add some filters on the getContacts - then look in API console to see how the query string is formatted.

I found in the PHP SDK the filters were not being passed as comma separated and resulted in them being ignored.

**Comment by SerKnight** [comment]
Hi @blairjo  Fixed in `4.1.3`
> https://github.com/XeroAPI/xero-node/pull/341/files#diff-fc0c29dfabaca58a40fb2861abf6ca8fR457

Also added a full `filtering / where` sample call in the sample app! Thanks for reporting this issue - much appreciated 🥳
> https://github.com/XeroAPI/xero-node-oauth2-app/blob/master/src/app.ts#L927
